Client/Frontend/Browser/Tabs/InactiveTabs/InactiveTabHeader.swift

```diff
@@ -30,9 +30,8 @@ class InactiveTabHeader: UITableViewHeaderFooterView, LegacyNotificationThemeabl
     lazy var titleLabel: UILabel = .build { titleLabel in
         titleLabel.text = self.title
         titleLabel.textColor = UIColor.legacyTheme.homePanel.activityStreamHeaderText
-        titleLabel.font = DynamicFontHelper.defaultHelper.preferredFont(
-            withTextStyle: .headline,
-            maxSize: 17)
+        titleLabel.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .headline,
+                                                                        size: UX.descriptionFontSize)
```

Comment:
@nbhasin2 we can't fix this warning like this. This needs to be fixed with a [specific task](https://mozilla-hub.atlassian.net/browse/FXIOS-5320) and requires constraints changes. The warning that was causing the build failure is fixed on main. You can rebase with main, and revert this change pls.

Reply:
Hahah, ok good to know! I'll try to be slower next time ;P